AiNga04/Ecommerce
# Add dark mode support to user dashboard

Description
This PR introduces a dark mode feature for the user dashboard. Users can now toggle between light and dark themes using a button in the top-right corner.

Fixes #45.

Changes Made

Added a toggle button for theme switching.
Updated CSS files to support both light and dark themes.
Persisted user preference using local storage.
Steps to Test

Go to the user dashboard.
Click the "Dark Mode" toggle button.
Verify the theme changes immediately.
Refresh the page and ensure the selected theme is retained.
